prisma/schema.prisma

```diff
@@ -49,6 +49,7 @@ model User {
   credentials   Credential[]
   teams         Membership[]
   bookings      Booking[]
+  selectedCalendars      SelectedCalendar[]
```

Comment:
I'm thinking, maybe it makes sense to just call this "Calendar"?

Reply:
With that name I wanted to express that the list does not contain a list of all available calendars (which would require some sort of syncing with all integrations), but only the selected ones.